studio/src/editors/variant-picker.js

```diff
@@ -1,13 +1,5 @@
 import { html, LitElement } from 'lit';
-
-//TODO make that feed (excepts ALL maybe) dynamically served from milo
-const VARIANTS = [
-    { label: 'All', value: 'all', surface: 'all' },
-    { label: 'Catalog', value: 'catalog', surface: 'acom' },
-    { label: 'Slice', value: 'ccd-slice', surface: 'ccd' },
-    { label: 'Special offers', value: 'special-offers', surface: 'acom' },
-    { label: 'Suggested', value: 'ccd-suggested', surface: 'ccd' },
-];
+import { VARIANTS } from '../constants.js';
```

Comment:
IMO variants can style in `variant-picker.js`

Reply:
Done